script.js

```diff
@@ -0,0 +1,768 @@
+var board = [[0, 0, 0], [0, 0, 0], [0, 0, 0]];
+var round = 0;
+var computerplay = false;
+
+
+var clickedBox = function (element){
+	var turn = "";
+	if ((element.classList.contains('blue') === false)&& 
+		(element.classList.contains('yellow') === false)){
+		if (round % 2 === 0){
+			turn = "X";
+			round++;
+			element.classList.add("blue");
+					element.innerHTML = turn;
+			buildArray(element);
+			winner2();
+			if (computerplay === true){
+				computerturn();	
+		}else {
+			turn = "O";
+			round++;
+			element.classList.add("yellow");
+			element.innerHTML = turn;
+			buildArray(element);
+			winner2();
+			if (computerplay === true){
+				computerturn();	
+			} 
+		}
+	}
+}
+
+};
+
+var computerturn = function(){
+	if (round === 1){
+		x =compWin();
+		if (x === false){
+			y = blockMove();
+			if (y === false){
+				compMove();
+				}
+			}
+			winner2();
+		}else if ((round === 3) && (!((document.getElementById("winner").innerHTML === "X is the Winner!"))) &&
+		(!(document.getElementById("winner").innerHTML === "O is the Winner!"))){
+			x =compWin();
+			if (x === false){
+				y = blockMove();
+				if (y === false){
+					compMove();
+				}
+			}
+			winner2();
+		} else if ((round === 5) && (!((document.getElementById("winner").innerHTML === "X is the Winner!"))) &&
+		(!(document.getElementById("winner").innerHTML === "O is the Winner!"))){
+			x =compWin();
+			if (x === false){
+				y = blockMove();
+				if (y === false){
+					compMove();
+				}
+			}
+			winner2();	
+		} else if ((round === 7) && (!((document.getElementById("winner").innerHTML === "X is the Winner!"))) &&
+		(!(document.getElementById("winner").innerHTML === "O is the Winner!"))){
+			if (x === false){
+				y = blockMove();
+				if (y === false){
+					compMove();
```

Comment:
Since a lot of these `if` blocks perform the same code, it would be beneficial to call the code using a function, to make things more readable.